internal/humanitec_go/client/deltas.go

```diff
@@ -58,3 +58,77 @@ func (api *apiClient) CreateDelta(ctx context.Context, orgID, appID string, delt
 		return nil, fmt.Errorf("humanitec api: %s %s: HTTP %d - %s", req.Method, req.BaseURL, resp.StatusCode, http.StatusText(resp.StatusCode))
 	}
 }
+
+// GetDelta retrieves and existing Deployment Delta by ID.
+func (api *apiClient) GetDelta(ctx context.Context, orgID, appID string, deltaID string) (*humanitec.DeploymentDelta, error) {
+	apiPath := fmt.Sprintf("/orgs/%s/apps/%s/deltas/%s", orgID, appID, deltaID)
+	req := rest.Request{
+		Method:  http.MethodGet,
+		BaseURL: api.baseUrl + apiPath,
+		Headers: map[string]string{
+			"Authorization": "Bearer " + api.token,
+			"Accept":        "application/json",
+		},
+	}
+
+	resp, err := api.client.SendWithContext(ctx, req)
+	if err != nil {
+		return nil, fmt.Errorf("humanitec api: %s %s: %w", req.Method, req.BaseURL, err)
+	}
+
+	switch resp.StatusCode {
+
+	case http.StatusOK:
+		{
+			var res humanitec.DeploymentDelta
+			if err = json.Unmarshal([]byte(resp.Body), &res); err != nil {
+				return nil, fmt.Errorf("humanitec api: %s %s: parsing response: %w", req.Method, req.BaseURL, err)
+			}
+			return &res, nil
+		}
+
+	default:
+		return nil, fmt.Errorf("humanitec api: %s %s: HTTP %d - %s", req.Method, req.BaseURL, resp.StatusCode, http.StatusText(resp.StatusCode))
+	}
+}
+
+// UpdateDelta updates an existing Deployment Delta for the orgID and appID with the given deltaID.
+// The Deltas in the request will be combined and applied on top of existing Delta to produce a merged result.
+func (api *apiClient) UpdateDelta(ctx context.Context, orgID string, appID string, deltaID string, deltas []*humanitec.UpdateDeploymentDeltaRequest) (*humanitec.DeploymentDelta, error) {
+	data, err := json.Marshal(deltas)
+	if err != nil {
+		return nil, fmt.Errorf("marshalling payload into JSON: %w", err)
+	}
+
+	apiPath := fmt.Sprintf("/orgs/%s/apps/%s/deltas/%s", orgID, appID, deltaID)
+	req := rest.Request{
+		Method:  http.MethodPatch,
+		BaseURL: api.baseUrl + apiPath,
+		Headers: map[string]string{
+			"Authorization": "Bearer " + api.token,
+			"Content-Type":  "application/json",
+			"Accept":        "application/json",
+		},
+		Body: data,
+	}
+
+	resp, err := api.client.SendWithContext(ctx, req)
+	if err != nil {
+		return nil, fmt.Errorf("humanitec api: %s %s: %w", req.Method, req.BaseURL, err)
+	}
+
+	switch resp.StatusCode {
+
+	case http.StatusOK, http.StatusCreated:
```

Comment:
Does the `PATCH` request ever return http.StatusCreated?

Reply:
nope, just 200's